### PR TITLE
dependency updates

### PR DESCRIPTION
**Pull request by jaredperreault-okta**

### CODE REVIEW

File: samples/templates/webpack-spa/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "prepare": "yarn build",
     "build": "webpack",
-    "start": "webpack-dev-server",
+    "start": "npx webpack serve",
```
Comment (aarongranick-okta) [inline samples/templates/webpack-spa/package.json:9]: why use npx?

File: samples/templates/webpack-spa/webpack.config.js [hunk 2]
```diff
@@ -36,15 +36,15 @@ module.exports = {
...
       path.join(__dirname, 'public'),
       SIW_DIR
     ],
     port: PORT,
-    before: function(app) {
-      app.get('{{ redirectPath }}', redirectToOrigin);
-      app.get('/login', redirectToOrigin);
-      app.get('/profile', redirectToOrigin);
+    onBeforeSetupMiddleware: function(devServer) {
+      devServer.app.get('/login/callback', redirectToOrigin);
```
Comment (aarongranick-okta) [inline samples/templates/webpack-spa/webpack.config.js:45]: is there a reason we can't support the template here?

File: test/apps/app/package.json [hunk 3]
```diff
@@ -38,6 +38,9 @@
   "peerDependencies": {
     "@okta/okta-signin-widget": "^6.0.0"
   },
+  "resolutions": {
+    "follow-redirects": "^1.14.9"
```
Comment (aarongranick-okta) [inline test/apps/app/package.json:42]: which dependency is this related to?

File: samples/templates/webpack-spa/webpack.config.js (hunk 2 above)
Comment (jaredperreault-okta) [reply samples/templates/webpack-spa/webpack.config.js:45]: Not sure I understand the question. This syntax changed due to a major version bump.

https://github.com/okta/okta-auth-js/pull/1218/files#diff-25616d4e5ff9af120aa428e51aeaaa20505440f3e3945373607ba8b72cd35292R26

https://github.com/webpack/webpack-dev-server/blob/master/CHANGELOG.md#-breaking-changes-4

> before, after and setup were removed in favor onBeforeSetupMiddleware (previously before) and onAfterSetupMiddleware options (previously after)

File: test/apps/app/package.json (hunk 3 above)
Comment (jaredperreault-okta) [reply test/apps/app/package.json:42]: `http-proxy-middleware`

https://github.com/okta/okta-auth-js/pull/1218/files#diff-26790002770945af943b6adbd98e2dcde649575b87198392113c3301ba5fcd8cR26

File: samples/templates/webpack-spa/webpack.config.js (hunk 2 above)
Comment (jaredperreault-okta) [reply samples/templates/webpack-spa/webpack.config.js]: nevermind, fixed

File: test/apps/react-mfa-v1/package.json [hunk 4]
```diff
@@ -4,24 +4,20 @@
...
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.3",
-    "web-vitals": "^1.0.1"
+    "react-router-dom": "^5.2.0"
   },
   "devDependencies": {
-    "react-app-rewired": "^2.1.8",
-    "webpack": "^4.46.0"
+    "@vitejs/plugin-react": "^1.0.7",
+    "vite": "^2.8.0"
```
Comment (denysoblohin-okta) [inline test/apps/react-mfa-v1/package.json:13]: 👍

File: samples/templates/webpack-spa/webpack.config.js (hunk 2 above)
Comment (aarongranick-okta) [reply samples/templates/webpack-spa/webpack.config.js]: I mean the template `{{ redirectPath }}`

**Review by denysoblohin-okta — APPROVED** [review]

File: samples/templates/webpack-spa/webpack.config.js (hunk 2 above)
Comment (jaredperreault-okta) [reply samples/templates/webpack-spa/webpack.config.js]: Yea, this was fixed: https://github.com/okta/okta-auth-js/pull/1218/files#diff-517e04b9fefa77f6361bc25d43ae5ce8af3cd841caa15f73b3ac5eca247193e9R45
